ui-integration-test-lib/src/androidTest/java/co/electriccoin/zcash/ui/integration/test/Global.kt

```diff
@@ -0,0 +1,63 @@
+package co.electriccoin.zcash.ui.integration.test
+
+import android.content.Context
+import android.os.Build
+import androidx.annotation.StringRes
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiObject
+import androidx.test.uiautomator.UiSelector
+
+fun getStringResource(@StringRes resId: Int) = ApplicationProvider.getApplicationContext<Context>().getString(resId)
+
+fun getStringResourceWithArgs(@StringRes resId: Int, vararg formatArgs: String) = ApplicationProvider.getApplicationContext<Context>().getString(resId, *formatArgs)
+
+// We're using indexes to find the right button, as it seems to be the best available way to test a click
+// action on a permission button. These indexes remain the same for LTR as well as RTL layout direction.
+fun getPermissionNegativeButtonUiObject(): UiObject? {
+    if (Build.VERSION.SDK_INT < 23) {
```

Comment:
Do we need this check if our min is 23 or 24+?

Reply:
We don't. I was thinking about not having it too. I used it just for clarity. Removing it.